chcemvediet/apps/accounts/forms.py

```diff
@@ -99,10 +100,14 @@ class SettingsForm(forms.Form):
             )
 
     days_to_publish_inforequest = forms.IntegerField(
-            min_value=0,
-            max_value=MAX_DAYS_TO_PUBLISH_INFOREQUEST,
             label=_(u'accounts:SettingsForm:days_to_publish_inforequest:label'),
             help_text=_(u'accounts:SettingsForm:days_to_publish_inforequest:help_text'),
+            widget=RangeWidget(attrs={
+                u'class': u'form-control-range',
```

Comment:
Zvykneme classy, co su "lokalne" ku projektu prefixovat "chv-"

Btw, preco tato classa je tu a nie priamo vo widgete? Dava zmysel, aby sa pri inych pouzitiach niekde customizovala? Vobec treba aby to malo classu? Nestaci v sass proste pouzit `> input`?

Classy navyse posielane do konstruktora widgetu sluzia hlavne na rozlisenie roznych sposobov vyrendrovania toho widgetu na roznych miestach. Potom by sa classou povedalo, ze ktore vyrendrovanie chceme pouzit.

Reply:
Nie netreba, riesil som to cez classu ako jeden zo sposob vyrendovania.